examples/document-search/multimodal.py

```diff
@@ -0,0 +1,73 @@
+# /// script
```

Comment:
Please add description to this example, should follow the format from [chroma_otel.py](https://github.com/deepsense-ai/ragbits/blob/main/examples/document-search/chroma_otel.py).
> I think we should start adding more comments to the examples. It would be sufficient to write a little summary / docstring on module level explaining what we're doing in the script

_Originally posted by @mhordynski in https://github.com/deepsense-ai/ragbits/pull/168#discussion_r1830614524_

Reply:
Done